pkg/trace/README.md

```diff
@@ -19,3 +23,25 @@ trace_push_batch_size = 1000
 # comma separate string. For example: "consensus_round_state,mempool_tx".
 tracing_tables = "consensus_round_state,mempool_tx"
 ```
+
+To retrieve a table remotely using the pull based server, call the following
+function:
+
+```go
+err := GetTable("http://1.2.3.4:26661", "mempool_tx", "directory to store the file")
+if err != nil {
+    return err
+}
+```
+
+This will download and save the file to the specified directory in the
+`table_name.jsonl` format.
```

Comment:
[no change needed] TIL there is a JSONL format. Ref: https://jsonlines.org/

Reply:
it;s pretty neat